plugins/TestRunner/Commands/SyncScreenshots.php

```diff
@@ -60,30 +63,31 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $buildNumber = $input->getArgument('buildnumber');
+        $agent = $input->getOption('agent');
         $screenshotsRegex = $input->getArgument('screenshotsRegex');
         $repository = $input->getOption('repository');
         $httpUser = $input->getOption('http-user');
         $httpPassword = $input->getOption('http-password');
 
-        $screenshots = $this->getScreenshotList($repository, $buildNumber, $httpUser, $httpPassword);
+        $screenshots = $this->getScreenshotList($repository, $buildNumber, $httpUser, $httpPassword, $agent);
 
         $this->logger->notice('Downloading {number} screenshots', array('number' => count($screenshots)));
         foreach ($screenshots as $name => $url) {
-            foreach ($screenshotsRegex as $regex) {
```

Comment:
Is there any good reason for removing the possibility to provide multiple regexes here?

Reply:
@sgiehl ops, that's me, revert it